build/zsh/files/system-zshrc

```diff
@@ -27,5 +27,6 @@
 #
```

Comment:
While in here, please drop the ident line and replace the licence with the new shorter version

```
#
# This file and its contents are supplied under the terms of the
# Common Development and Distribution License ("CDDL"), version 1.0.
# You may only use this file in accordance with the terms of version
# 1.0 of the CDDL.
#
# A full copy of the text of the CDDL should have accompanied this
# source. A copy of the CDDL is also available via the Internet at
# http://www.illumos.org/license/CDDL.
#
```